### ISSUE TITLE
On focus of the accessible buffer, if the last command failed, play audio cue

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175013]
seems like this would be useful based on how @jooyoungseo operates IE using the editor to formulate terminal input, running the selected text in the active terminal, and then focusing the accessible buffer

**Comment by meganrogge** [comment]
this works already
